src/components/Checkbox/__stories__/checkbox.stories.scss

```
@@ -37,12 +38,13 @@
   &_list {
     display: flex;
     flex-direction: column;
+    font-family :var(--font-family) !important;
```

`font-family` is not the only thing to define the font. Let's instead use `sb-content-text` mixin and increase the specificity of selector, so all the styles would apply. Btw, even thought, it's only documentation using `!important` in general is a bad practice

I propose we'd do smth like this:
```suggestion
    span {
      @include sb-content-text;
    }
```


And also we'll need to wrap `Use checkboxes to:` text with `span` tag
This way all out mixin styles will be applied both to the header title and to the list items without using `!important`